src/openms/source/ANALYSIS/TARGETED/PrecursorIonSelectionPreprocessing.cpp

```diff
@@ -331,10 +331,8 @@ namespace OpenMS
         digest.digest(aa_seq, vec);
 
         // enter peptide sequences in map
-        std::vector<AASequence>::iterator vec_iter = vec.begin();
-
         std::vector<String> peptide_seqs;
-        for (; vec_iter != vec.end(); ++vec_iter)
+        for (vector<AASequence>::iterator vec_iter = vec.begin(); vec_iter != vec.end(); ++vec_iter)
```

Comment:
great catch but this loop can be turned into a range for loop - which would be even better